gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "3.6.1",
+  "version": "3.6.2",
```

Comment:
Should be a minor version

Reply:
> Should be a minor version

I'm not always certain when it's a minor modification to an existing component: https://github.com/sonatype/sonatype-react-shared-components/pull/309/commits/3885453a58bba05042c563730145acc81412887b